stockfish_file_path: "/home/mitch/pkgs/stockfish/stockfish-ubuntu-x86-64-avx2"
stockfish_difficulty: 2
